README.md

```diff
@@ -217,6 +217,18 @@ The ViewModifiers are used to customise the look and feel of the BottomSheet.
 - Changing the threshold does not affect whether either option is enabled.
 - The default threshold is 30% (0.3).
 
+`.iPadFloatingSheet(Bool)`: Makes the iPad's bottom sheet be an actual bottom sheet when set to true.
+- The default is `false`.
+
+`.sheetWidth(BottomSheetWidth)`: Sets the width of the sheet.
+- Can be relative through `BottomSheetWidth.relative(x)`.
+- Can be absolute through `BottomSheetWidth.absolute(x)`.
+- Set to `BottomSheetWidth.platformDefault` to let the library decide the width.
+
+`.accountForKeyboardHeight(Bool)`: Adds bottom padding to the main content equal to the keyboard height when the keyboard appears.
+- The default is `false`.
+- Will not move the sheet further up if the sheet has a smaller height than the keyboard.
+- Not available on Mac.
```

Comment:
I make the documentation comment always the same as the text in the readme. So that it is easier to find all available information in one both places and that I can change them easily.